### ISSUE TITLE
"Map Source" operation in menu

### ISSUE THREAD

**Issue by andrew-davie** [osmandapp/Osmand#2412]
The "Map source" operation is inconsistent and confusing.
I believe it should have an on/off slider just like (say) the GPX track. When you slide it to on, you go to the screen explaining what it's all about. Once you come back you can click on map source and select the sources, just like GPX option.  It is very confusing for new users to realise you have to turn on map source to see aything, especially when it doesn't look like a "turn on" type thing in the menu.
TL;DR - make "map source" an on/off slider.

**Comment by vshcherb** [comment]
We are about to change this screen and we will take it into consideration. As I see you would like to have the same as for Underlay/Overlay. Though the argument against it that you always have selected Map Source cause you always see a map.
